src/Symfony/Component/Yaml/Parser.php

```diff
@@ -436,6 +436,12 @@ private function doParse($value, $flags)
                     $value = '';
 
                     foreach ($this->lines as $line) {
+                        if (empty(trim($line)) && !empty($line) && in_array($this->currentLine, ['{', '['])) {
```

Comment:
we try to avoid using `empty()`, can you please try *not* using them?

Reply:
Removed empty for '' === or '' !==.  Can I ask why Symfony does not use empty ?  Traps of false, null, et.al.?